client/layout/masterbar/masterbar-help-center/index.jsx

```diff
@@ -24,7 +31,13 @@ const MasterbarHelpCenter = ( { tooltip } ) => {
 		helpCenterVisible: select( HELP_CENTER_STORE ).isHelpCenterShown(),
 		unreadCount: select( HELP_CENTER_STORE ).getUnreadCount(),
 	} ) );
-	const { setShowHelpCenter } = useDataStoreDispatch( HELP_CENTER_STORE );
+	const { setShowHelpCenter, setNavigateToRoute } = useDataStoreDispatch( HELP_CENTER_STORE );
+
+	// Check if the new menu panel feature is enabled (both feature flag AND query param must be true)
+	const queryParams = parse( window.location.search.replace( /^\?/, '' ) );
+	const isMenuPanelEnabled =
+		config.isEnabled( 'help-center-menu-panel' ) &&
+		queryParams[ 'help-center-menu-panel' ] === 'true';
```

Comment:
Why do we need both? We can maybe set this flag to false and then test by passing `flags=help-center-menu-panel` instead.

```suggestion
	// Check if the new menu panel feature is enabled (both feature flag AND query param must be true)
	const isMenuPanelEnabled =  config.isEnabled( 'help-center-menu-panel' );
```

Reply:
Done! I'll fix the z-index tomorrow